apps/src/music/player/MusicLibrary.ts

```diff
@@ -433,10 +446,17 @@ export interface SoundData {
   skipLocalization?: boolean;
 }
 
+export interface ImageAttributionCopyright extends ImageAttribution {
+  artist: string;
+}
+
 export interface ImageAttribution {
   author: string;
   color?: string;
   position?: 'left' | 'right';
+  src?: string;
+  licenseVersion: 'CC2' | 'CC3' | 'CC4' | 'C';
```

Comment:
For my understanding, what does 'C' refer to?

Edit: seems like copyright vs Creative Commons?

Reply:
Yes, it's just a shorthand for a more regular copyright message.